documentation/assemblies/upgrading/assembly-downgrade.adoc

```diff
@@ -0,0 +1,26 @@
+// This assembly is included in the following assemblies:
+//
+// deploying/master.adoc
+
+[id='assembly-downgrade-{context}']
+= Downgrading Strimzi
+
+If you are encountering issues with the latest version of Strimzi,
```

Comment:
The `latest` version of Strimzi sounds weird given the documentation is for the `latest` Strimzi version for a month or two but then it is for some older version but might be still use. Can we maybe rephrase it to say something like _If you are encountering issues with the version of Strimzi you upgraded to_?